src/index.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Lato&family=Poppins&display=swap');
+
```

Comment:
- [ ] The design aim for the app was to ensure that we at least make it look like a mobile app. Whilst the current implementation looks good on the Desktop, we still have to ensure that the implementation of the provided design guideline is achieved. Currently, this is the look of our app when opened in mobile view 👇🏼:
  
  ![image](https://user-images.githubusercontent.com/45002654/175386426-3e548c65-1884-4658-ac7c-e67855abf5cb.png)
  
  As evident above, elements aren't properly aligned and are overflowing the viewport width. Our homepage should at least look like this 👇🏼:  
  
  ![image](https://user-images.githubusercontent.com/45002654/175386754-e29f0e03-b83f-4036-a7b9-578d6a4eb89b.png)
  
  Let's also work on the details page, currently, it has elements overlapping each other 👇🏼:
  
  ![image](https://user-images.githubusercontent.com/45002654/175387236-edfcf6cb-261d-41cd-873f-8bfb408ae73f.png)
  
  It would be nice If you worked with the design that was given for the details page. In the design, we see the statistics(in your case, cases per country) ordered one after the other from top to bottom and not side to side. Please work on this too. Let the details page look like this 👇🏼:
  
  ![image](https://user-images.githubusercontent.com/45002654/175387850-b8577ca8-5f5f-4a24-9ca6-2d37c9b4993c.png)
  
  NOTE: If you would like to know that you are implementing the expected design, always test the pages in mobile view.

Reply:
Changes implemented